exercises/spiral-matrix/spiral_matrix.py

```diff
@@ -0,0 +1,2 @@
+def spiral():
```

Comment:
It was reported here https://github.com/exercism/python/issues/509 that "placeholder" files should have expected parameters to help users start working on exercises quicker